plainly-plugin/src/ui/components/projects/LinkedProject.tsx

```diff
@@ -108,7 +108,7 @@ export function LinkedProject({
                 <Tooltip text="Sync status">
                   <FolderSync className="size-3" />
                 </Tooltip>
-                <p>Local v{projectData?.revisionCount}</p>
+                <p>Local v{plainlyProject?.revisionCount}</p>
```

Comment:
if this is undefined we will see "Local vundefined". Should we use `0` as default or render <p> conditionally